test/evergreen.yml

```diff
@@ -2886,7 +2833,7 @@ tasks:
             set -o errexit
```

Comment:
Can we get rid of `configure_env_vars` [here](https://github.com/wiredtiger/wiredtiger/pull/7317/files#diff-42f20e21217bd7d4b14489056e1ea24ea2b87599c85413d49182a3b0b721c132R2826)?

Reply:
Good point, fixed in: 47a822a